src/ispyb/sqlalchemy/__init__.py

```diff
@@ -8,8 +8,514 @@
 import sqlalchemy.orm
 from sqlalchemy.orm import relationship
 
-from ._auto_db_schema import *  # noqa F403; lgtm
-from ._auto_db_schema import AutoProcProgram, AutoProcScaling, ProcessingJob
+from ._auto_db_schema import (
+    CTF,
+    PDB,
```

Comment:
Okay, this is a huge change and needs an accompanying change in the database schema update code to automatically maintain these statements. Otherwise this will hurt us in the future and cause the model and schema to diverge. I would suggest leaving out these changes and implementing the code first (unless you have done so already?)